site/gatsby-site/src/components/discover/QuickAccess.js

```diff
@@ -22,7 +22,7 @@ export default function DiscoverQuickAccess({ className }) {
     <Wrapper className={`${className} p-4 md:bg-text-light-gray h-24`}>
       <LocalizedLink
         to="/apps/discover"
-        className="fs-5 tw-btn text-center shadow w-1/2 h-16 bg-white hover:bg-primary-blue hover:text-white"
+        className="fs-5 tw-btn text-center shadow w-1/2 h-16 bg-white hover:bg-light-orange hover:text-white"
```

Comment:
While it fits with the other sidebar items, I think orange looks a bit weird for buttons, especially since we don't have any other orange buttons. But it's a matter of opinion. I would suggest trying it with the blue we get with the Flowbite buttons, and if you like that less, I'm fine with orange too.

Reply:
I agree with Luna, tell us what you think @clari182